deploy/roles/guest/tasks/main.yml

```diff
@@ -17,21 +17,8 @@
     depth: "{{ git_clone_depth | default(omit) }}"
     force: yes
 
-- name: Copy BKC TDX guest kernel config
-  copy:
-    src: "{{ playbook_dir | dirname }}/bkc/kafl/linux_kernel_tdx_guest.config"
-    dest: "{{ guest_root }}/.config"
-
-- name: Enable Virtio harness
-  command: "{{ item }}"
-  args:
-    chdir: "{{ guest_root }}"
-  with_items:
-    - ./scripts/config -e CONFIG_TDX_FUZZ_KAFL
-    - ./scripts/config -e CONFIG_TDX_FUZZ_HARNESS_DOINITCALLS_VIRTIO
-
-- name: Build TDX guest kernel
-  make:
-    chdir: "{{ guest_root }}"
-    params:
-      --jobs: "{{ ansible_processor_nproc }}"
+- name: Prepare fuzzing build of TDX guest kernel
+  include_tasks: build.yml
```

Comment:
I just tested your PR, and I realized that the tasks in `build.yml` were not executed.

The solution is to replace `include_tasks` with [`import_tasks`](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/import_tasks_module.html) which is evaluated statically before the playbook is executed.

Reply:
I could not reproduce, this seemed to work fine with either option:
`make deploy --  --tags guest,guest_build`

But indeed, import_tasks seems more appropriate when just splitting up playbooks for convenience.